Makefile

```diff
@@ -23,6 +23,17 @@ GOPROXY=direct
 GOPATH=$(shell go env GOPATH)
 GOOS=$(shell go env GOOS)
 GOBIN=$(shell pwd)/bin
+GO ?=go
+deps_diff := diff --no-dereference -N
+
+# $1 - temporary directory
+define restore-deps
+	ln -s $(abspath ./) "$(1)"/current
+	cp -R -H ./ "$(1)"/updated
+	$(RM) -r "$(1)"/updated/vendor
+	cd "$(1)"/updated && $(GO) mod tidy && $(GO) mod vendor && $(GO) mod verify
```

Comment:
shouldn't we run `go mod tidy`  after `go mod vendor`?

Reply:
Adjusted this to be after `go mod vendor`.